docs/development_suite/api-console/api-design/services.md

```diff
@@ -104,31 +107,55 @@ For a step-by-step guide on creating a microservice using a Docker image, refer
+- **Clone**: this button, present only in microservices created from Examples and Templates, enables to clone code repository directly from Console. The code repository can be copied with both `ssh` and `https`.
 
-The detail of each microservice is divided in the following sections:
+The following paragraphs explain each section of the Microservice detail page.
 
 ### Microservice
 
 In this section, you can review and update some base information about the microservice, like the description and the API documentation path.
 
-In case your microservice has been created manually from a Docker Image or from a Marketplace Template/Example, you can also configure the *Docker Image Name*.
+#### Microservice Details
 
-![service-detail-new](img/service-detail-new.png)
+The Details card contains and lets you manage information such as the version, the docker image and the description of the microservice.
 
-If the microservice has been created from a versioned Marketplace plugin, the *Docker Image Name* is shown in *read-only* mode and cannot be changed, since the it is managed by the Marketplace plugin itself.
+In case your microservice has been created from a Plugin, the *Docker Image Name* is shown in *read-only* mode and cannot be changed, since the it is managed by the Plugin itself.
 
 ![Detail of a service generated from a versioned Marketplace plugin](img/versioned-service.png)
 
+Otherwise, if your microservice has been created manually from a Docker Image or from a Marketplace Template/Example, you can also configure the *Docker Image Name* through a dedicated input field.
+
+![service-detail-new](img/service-detail-new.png)
+
+The *Docker Image Name* is the name of the Docker image that will be used to deploy the microservice. It is composed by the container registry hostname, the image name and the tag.
+
+The container registry hostname can be changed with a dropdown list, which contains the [Container Registries configured in the Project Settings](/console/project-configuration/project-settings.md#container-registries).
+
+![Docker Image input field](img/microservices-docker-image-field.png)
+
+The image name and the tag can be manually inserted in the dedicated input field.
+
+:::caution
+
+Starting from version v13.5.0, the container registry hostname is not part anymore of the Docker Image Name manual input field, but it can only be selected from the dropdown list.
+
+If you need to use a container registry hostname that is not present in the dropdown list, you have to create it in the Project Settings, or contact your Project Administrator or Company Owner to do it for you.
+
+In case a service already uses a container registry not in the list, an informative badge will advise the user to create it, or to choose a new one from the list. The project can still be saved with the unknown container registry, to avoid disruption.
```

Comment:
```suggestion
In case a service already uses a container registry not in the list, an informative badge will suggest the user to create it, or to choose a new one from the list. The project can still be saved with the unknown container registry, to avoid disruption.
```